src/unittest/test.cpp

```diff
@@ -343,6 +343,9 @@ Markdown lol \\|\\|spoiler\\|\\| \\~\\~strikethrough\\~\\~ \\`small \\*code\\* b
 
 		set_test(USER_GET_CREATION_TIME, false);
 		set_test(USER_GET_CREATION_TIME, (uint64_t)user1.get_creation_time() == 1465312605);
+
+		set_test(USER_GET_URL, false);
+		set_test(USER_GET_URL, user1.get_url() == dpp::utility::url_host + "/users/189759562910400512");
```

Comment:
I feel this should be changed to `set_test(USER_GET_URL, user1.get_url() == (dpp::utility::url_host + "/users/id"));`, this way it's easily readable what is expected.

I'd like more thoughts on this though!

Reply:
don`t really got what you mean, `dpp::utility::url_host + "/users/id"`  is impossible as unit test, string can not be assigned to dpp::snowflake...

Reply:
No no, replace `id` with the actual ID, i just didn't wanna type out the numbers lol. I just mean put brackets around the stuff after `==` haha

Reply:
well, tests for other urls are made same way, so it would be not nice to make one test in one way and other in another way.

https://github.com/brainboxdotcc/DPP/blob/fd3f87522a3647670e7f95afbf27b5ab92af0fe9/src/unittest/test.cpp#L417-L423

Reply:
> well, tests for other urls are made same way, so it would be not nice to make one test in one way and other in another way.

Hm, yeah, might be something to discuss in discord for a future clean-up PR or something, just to improve readability :)

Reply:
For now I will left it as it is